### PR TITLE
Reduce execution time of TestBuildHistory

### LINKED ISSUE #12671: Reduce execution time of TestBuildHistory
I think 26 steps is too much, 5 will be enough.

### PR DESCRIPTION
**Pull request by HuKeping**
Closes #12671

Signed-off-by: Hu Keping hukeping@huawei.com

### CODE REVIEW

File: integration-cli/docker_cli_history_test.go [hunk 1]
```diff
@@ -52,9 +31,9 @@ RUN echo "Z"`,
 	}
 
 	actualValues := strings.Split(out, "\n")[1:27]
```
Comment (runcom) [inline integration-cli/docker_cli_history_test.go]: I think this should be `[1:4]`

Comment (HuKeping) [inline integration-cli/docker_cli_history_test.go]: Thanks! maybe [1:6]?

Comment (runcom) [inline integration-cli/docker_cli_history_test.go]: haven't looked at the code :P just a thought

Comment (runcom) [inline integration-cli/docker_cli_history_test.go]: yup 1:6

**Comment by unclejack** [comment]
This is a bad idea. There was a good reason behind having that many RUN commands.

**Comment by duglin** [comment]
Perhaps we could modify the test with a comment to explain why it need to have that many RUNs?

**Comment by cpuguy83** [comment]
@duglin It has it already
>// This is a heisen-test.  Because the created timestamp of images and the behavior of
// sort is not predictable it doesn't always fail.

**Comment by duglin** [comment]
ah, I didn't expand the diff enough. thanks

**Comment by HuKeping** [comment]
so I think we should keep this it as  before ?

**Comment by unclejack** [comment]
@HuKeping The number of layers it had before was required to trigger this test previously. I'd very much prefer keeping it this way, rather than risk making it less useful and less likely to trigger the bug in case of a regression.

I think reducing the number of layers for this particular test is a bad idea and I also remember why this test was introduced, so it should be left alone.

Instead of attempting to speed up this test, perhaps we should look into making other changes to speed things up overall.

NOT LGTM

**Comment by cpuguy83** [comment]
Closing since this test should be left as is for now.

**Comment by LK4D4** [comment]
@unclejack You talking about another test. But okay if you're so against it.

**Pull request closed without merging**